config/locales/cy.yml

```diff
@@ -977,6 +977,143 @@ materials:
+  dashboard:
+    title: Eich cyfrif
+    sectionLinks:
+    - label: Newid eich cyfrinair
+      href: /user/password/reset
+    - label: Newid eich cyfeiriad e-bost
+      href: /user/update-email
+    - label: Gweld eich ceisiadau ariannu
+      href: /apply/awards-for-all
+  login:
+    title: Mewngofnodi
+    introduction: >
+      <p>Mae hwn yn ffurflen gais newydd. Os nad ydych wedi mewngofnodi yma eisoes, efallai na fydd gennych gyfrif â ni - a bydd angen <a href="/user/register">creu cyfrif newydd</a> i ymgeisio.</p>
+    actionLabel: Mewngofnodi
+    registerLabel: creu cyfrif
+    forgottenPasswordPrompt: Wedi anghofio eich cyfrinair?
+    credentialError: Nid yw eich enw defnyddiwr a chyfrinair yn gywir. Neu efallai nad oes gennych gyfrif, gan fod hwn yn ffurflen gais newydd – ceisiwch <a href="/user/register">greu cyfrif</a>.
+  register:
+    title: Creu cyfrif
+    actionLabel: Creu cyfrif
+    loginLabel: mewngofnodi
+    genericError: >
+      Roedd problem wrth greu eich cyfrif. Os ydych yn meddwl bod gennych gyfrif eisoes, ceisiwch
+      <a href="%s">ailosod eich cyfrinair</a>.
+  forgottenPassword:
+    title: Wedi anghofio eich cyfrinair?
+    emailExplanation: Rhowch y cyfeiriad e-bost sy’n gysylltiedig â’ch cyfrif.
+    resetConfirmation: Os yw’r cyfeiriad e-bost yn gywir byddwch yn derbyn cyfarwyddiadau drwy e-bost.
+    callToAction: Ailosod cyfrinair
+    loginLink: Wedi ailosod cyfrinair? <a href="%s">Mewngofnodi</a>
+    email:
+      subject: Ailosod y cyfrinair i’ch cyfrif gwefan Cronfa Gymunedol y Loteri Genedlaethol
+      body: >
+        <h1>Cronfa Gymunedol y Loteri Genedlaethol</h1>
+        <h2>Cais am ailosod cyfrinair</h2>
+
+        <p><strong>Rydym newydd dderbyn cais i newid cyfrinair i’r e-bost hwn</strong></p>
+        <h3><a href="%s">Newid eich cyfrinair</a></h3>
+
+        <p><strong>Os nad ydych yn dymuno ailosod eich cyfrinair</strong></p>
+        <p>nid oes rhaid i chi wneud dim. Anwybyddwch yr e-bost hwn.</p>
+
+        <p>Diolch,</p>
+        <p>Cronfa Gymunedol y Loteri Genedlaethol</p>
+  resetPassword:
+    title: Ailosod eich cyfrinair
+    introduction: >
+      <p>Dewiswch gyfrinair newydd i’ch cyfrif drwy ei gofnodi isod. Sicrhewch eich bod yn dewis cyfrinair cryf.</p>
+    errorMessage: Roedd problem wrth ddiweddaru eich cyfrinair – Trïwch eto
+    errorExpired: Roedd eich dolen ailosod cyfrinair yn annilys neu wedi dod i ben. <a href="/user/password/forgot">Trïwch eto</a>.
+    email:
+      subject: Rydych newydd newid eich cyfrinair
+      body: >
+        <h1>Cronfa Gymunedol y Loteri Genedlaethol</h1>
+        <h2>Rydych newydd newid eich cyfrinair</h2>
+
+        <p>Helo,</p>
+        <p>E-bost byr i roi gwybod eich bod wedi newid eich cyfrinair ar-lein yn llwyddiannus.</p>
+        <p>Os nad mai chi oedd hwn – ac nad ydych yn gwybod unrhyw beth am unrhyw newidiadau i gyfrinair – cysylltwch <a href="https://www.tnlcommunityfund.org.uk/contact">â ni</a> ar unwaith.</p>
```

Comment:
Since, the hyperlinks for this sentence was not underlined, I online translated and replaced the hyperlink text. Hence, might need reviewing.